pkg/dogstatsd/server.go

```diff
@@ -40,7 +40,7 @@ var (
 	dogstatsdPacketsLastSec          = expvar.Int{}
 
 	tlmProcessed = telemetry.NewCounter("dogstatsd", "processed",
-		[]string{"type", "state"}, "Count of service checks/events/metrics processed by dogstatsd")
+		[]string{"metric_type", "state"}, "Count of service checks/events/metrics processed by dogstatsd")
```

Comment:
metric_type is a bit ambiguous IMO, since it typically means gauge/count instead of metrics/events/service_checks. How about:

```suggestion
		[]string{"message_type", "state"}, "Count of service checks/events/metrics processed by dogstatsd")
```

Reply:
100% agree, I was struggling to find something better and I've considered message type but it was too much linked to the code to me; as you're also suggesting it I'll go with it 👍